### PR TITLE
update statefulstore and statefulservice example manifests

### PR DESCRIPTION
**Pull request by JArgente**
Update both manifests so the "Run on GKE" section of documentation works properly

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @JArgente,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

**Comment by lightbend-cla-validator** [comment]
Hi @JArgente,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

File: samples/js-shopping-cart/js-shopping-cart.yaml [hunk 1]
```diff
@@ -3,14 +3,16 @@ apiVersion: cloudstate.io/v1alpha1
...
+    # Name of a deployed Datastore to use.
+    name: cassandra
+    # Datastore specific configuration
+    config:
+      # Since the datastore is a Cassandra datastore, we need to specify the keyspace to use.
+      keyspace: shoppingcart
   containers:
-    - image: cloudstateio/samples-js-shopping-cart:latest
-      name: user-function
+    - image: calncal/js-shopping-cart:latest
```
Comment (marcellanz) [inline samples/js-shopping-cart/js-shopping-cart.yaml:18]: `calncal/`
@JArgente this might be a missed leftover of local tests

Comment (JArgente) [reply samples/js-shopping-cart/js-shopping-cart.yaml:18]: @marcellanz You are right!, thanks for the suggestion

**Comment by marcellanz** [comment]
@JArgente please add @pvlugter from the core team for this review, thanks.

**Comment by JArgente** [comment]
> @JArgente please add @pvlugter from the core team for this review, thanks.

@marcellanz How can I add @pvlugter? I can't add reviewers or asign it to anyone.

**Comment by marcellanz** [comment]
> @marcellanz How can I add @pvlugter? I can't add reviewers or asign it to anyone.

`ping -c 1` @pvlugter

**Comment by JArgente** [comment]
`ping -c 1 @pvlugter`

File: samples/js-shopping-cart/cassandra-store.yaml [hunk 2]
```diff
@@ -6,9 +6,9 @@ kind: StatefulStore
 metadata:
   name: cassandra
 spec:
-
   # This is a Cassandra deployment, so supply configuration for Cassandra.
-  cassandra:
-
+  type: Cassandra
+  deployment: Unmanaged
```
Comment (pvlugter) [inline samples/js-shopping-cart/cassandra-store.yaml:11]: This was just changed recently for the new operator:

https://github.com/cloudstateio/cloudstate/pull/441/files#diff-bd08628341d8901dc8500e3f54739f4dL13-L20

File: samples/js-shopping-cart/js-shopping-cart.yaml [hunk 3]
```diff
@@ -3,14 +3,16 @@ apiVersion: cloudstate.io/v1alpha1
...
-        # Name of a deployed Datastore to use.
-        name: inmemory
-
+  datastore:
+    # Name of a deployed Datastore to use.
+    name: cassandra
+    # Datastore specific configuration
+    config:
+      # Since the datastore is a Cassandra datastore, we need to specify the keyspace to use.
+      keyspace: shoppingcart
```
Comment (pvlugter) [inline samples/js-shopping-cart/js-shopping-cart.yaml:16]: This was also changed for the new operator:

https://github.com/cloudstateio/cloudstate/pull/425/files#diff-01dc38e97c1ec4b24976edc69721cba2L9-L18

**Review by pvlugter — COMMENTED** [review]
Thanks @JArgente for looking at this. Sorry about the confusion. These files on master have been updated recently for a new operator. The previous versions for stable release can be found here:

https://github.com/cloudstateio/cloudstate/blob/v0.5.1/samples/js-shopping-cart/cassandra-store.yaml

https://github.com/cloudstateio/cloudstate/blob/v0.5.1/samples/js-shopping-cart/js-shopping-cart.yaml

I see that the references to master are from the README — so we should look at removing the outdated doc content from the README and include everything in the actual docs, and only reference the stable version.

(also pinging @jroper for any extra comments)

**Comment by JArgente** [comment]
Thanks for the explanation @pvlugter , so I suppose this PR should be closed.

**Comment by pvlugter** [comment]
Yes, let's close this PR. @JArgente, if you're interested in adding an up-to-date `Run on GKE` section to the docs instead, that would be great.

**Pull request closed without merging**